apps/electionguard-bindings/src/lib.rs

```diff
@@ -0,0 +1,180 @@
+use eg::{
+    ballot::BallotEncrypted,
+    contest_selection::ContestSelection,
+    device::Device,
+    election_manifest::{ContestIndex, ElectionManifest},
+    election_parameters::ElectionParameters,
+    election_record::PreVotingData,
+    guardian_public_key::GuardianPublicKey,
+    hashes::Hashes,
+    hashes_ext::HashesExt,
+    joint_election_public_key::JointElectionPublicKey,
+};
+use rand::{thread_rng, RngCore};
+use serde::{Deserialize, Serialize};
+use std::{collections::BTreeMap, convert::TryFrom};
+use tsify::Tsify;
+use util::csprng::Csprng;
+use wasm_bindgen::prelude::*;
+
+/// The missing typescript types for election guard structs.
```

Comment:
Yes. I thought as we want to feed our implementation back into the original election guard repository, it would be weird to add the necessary trait derivisions there (even if behind a feature flag?)

What's your opinion on this?

Reply:
Yeah, I think that's reasonable.